bindings/python/iota_sdk/types/address.py

```diff
@@ -69,19 +70,6 @@ class NFTAddress(Address):
     type: int = field(default_factory=lambda: int(AddressType.NFT), init=False)
 
 
-@json
-@dataclass
-# pylint: disable=function-redefined
-# TODO: Change name
-class AccountAddress():
```

Comment:
Remove? Not rename?

Reply:
Yes, because we will not have it anymore with the single address wallet